connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -520,6 +538,8 @@ async function syncOneFile(
       },
       "Skipping empty document"
     );
+
+    return false;
```

Comment:
Looks like it was missing from https://github.com/dust-tt/dust/pull/3173.

Reply:
Oh good catch!